event_log.c

```diff
@@ -0,0 +1,246 @@
+/*
+ * Copyright (c) 2020 Oracle and/or its affiliates. All rights reserved.
```

Comment:
Is this a copy of the log code from Secure Launch? If you made modifications, you should add yourself at least as an author.

Reply:
Done